### PR TITLE
Add link for post in list post

### PR DESCRIPTION
**Pull request by at-minhdao**
## Description
- Add link for post in list post
## Reference Requirements
- Product Backlog ID: PBI-10
- Task ID: #10.1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
- None

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
- Non-SQL
- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/backend/posts/index.blade.php [hunk 1]
```diff
@@ -29,13 +29,13 @@
...
-                    <th class="text-center" width="30%">
+                    <th class="text-left" width="30%">
                       {{ __('posts.short_content') }}
                     </th>
-                    <th class="text-center" width="10%">
+                    <th class="text-left" width="10%">
                       {{ __('posts.status') }}
                     </th>
-                    <th width="18%" class="text-center">
+                    <th class="text-left" width="18%">
```
Comment (at-hieule) [inline resources/views/backend/posts/index.blade.php]: Text ở trong `th` canh giữa hay canh trái đẹp hơn hì.Nếu fix vậy cho cái img UI vô PR để xem với

File: resources/views/backend/posts/index.blade.php [hunk 2]
```diff
@@ -53,8 +53,8 @@
                   @foreach ($posts as $post)
                     <tr>
                       <td class="text-center">{{ $post->id }}</td>
-                      <td>{!! Str::words($post->content, config('define.posts.size_short_content'),config('define.posts.three_dots'))  !!}</td>
-                      <td class="text-center">
+                      <td><a href="{{ route('posts.show', $post->id) }}">{!! Str::words($post->content, config('define.posts.size_short_content'),config('define.posts.three_dots'))  !!}</a></td>
```
Comment (at-hieule) [inline resources/views/backend/posts/index.blade.php]: `{!!` mở 1 space mà sao đóng 2 space vậy

**Comment by at-hieule** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**